addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -271,7 +271,12 @@ msgctxt "#31048"
 msgid "Available"
 msgstr ""
 
-#empty strings from id 31049 to 31051
+#empty string with id 31049
+
+#: /xml/Variables.xml
+msgctxt "#31050"
+msgid "Press [B]i[/B] to display plot, or long press [B]i[/B] to display player info"
```

Comment:
i is the icon on the button, not keyboard. Do you find it less confusing if we simply drop the <b>I</b> part?

Reply:
> i is the icon on the button, not keyboard.

Ah, what about "info button" then, to denote its functionality not its (current) visual appearance - otherwise if somebody redesigns the icon not to show just an i, this text would need to be adapted and i bet this would be overseen in the first place. ;-)

Reply:
done